### PR TITLE
Add support for f29 and remove f27

### PR DESCRIPTION
**Pull request by p3ck**

### CODE REVIEW

File: config/Dockerfiles/Jenkinsfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ executionID = UUID.randomUUID().toString()
 // Pod name to use
 c7podName = 'lp_centos7-' + executionID
 f28podName = 'lp_fedora28-' + executionID
-f27podName = 'lp_fedora27-' + executionID
+f27podName = 'lp_fedora29-' + executionID
```
Comment (14rcole) [inline config/Dockerfiles/Jenkinsfile]: f27podName?

File: config/Dockerfiles/Jenkinsfile [hunk 2]
```diff
@@ -179,9 +179,9 @@ stage('Tests') {
                                 args: '${computer.jnlpmac} ${computer.name}',
                                 command: '',
                                 workingDir: '/workDir'),
-                            containerTemplate(name: 'fedora27',
+                            containerTemplate(name: 'fedora29',
                                 alwaysPullImage: true,
-                                image: DOCKER_REPO_URL + '/' + OPENSHIFT_NAMESPACE + '/fedora27:' + FEDORA27_TAG,
+                                image: DOCKER_REPO_URL + '/' + OPENSHIFT_NAMESPACE + '/fedora29:' + FEDORA27_TAG,
```
Comment (14rcole) [inline config/Dockerfiles/Jenkinsfile]: FEDORA27_TAG?

File: config/Dockerfiles/Jenkinsfile (hunk 1 above)
Comment (p3ck) [reply config/Dockerfiles/Jenkinsfile]: doh!

File: config/Dockerfiles/fedora29/Dockerfile [hunk 3]
```diff
@@ -26,7 +26,7 @@ RUN dnf -y install python-pip ansible curl gcc python-devel python2-shade \
 
 RUN curl -o /etc/yum.repos.d/beaker-client.repo \
             https://beaker-project.org/yum/beaker-client-Fedora.repo; \
-    yum install -y beaker-client; \
+    yum install beaker-client; \
```
Comment (14rcole) [inline config/Dockerfiles/fedora29/Dockerfile]: Why does this no longer need the -y flag?

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[merge]

**Pull request merged**